bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/bhros/SeqConfigBHROS.java

```diff
@@ -18,6 +21,9 @@
 
     public static final SPComponentType SP_TYPE = SPComponentType.ITERATOR_BHROS;
 
+    public static final ISPNodeInitializer<ISPSeqComponent, SeqConfigBHROS> NI =
```

Comment:
All these reductions down to `NI` and elimination of subclasses is very nice indeed.